GliaWidgets/SecureConversations/SecureConversations.Availability.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+extension SecureConversations {
+    struct Availability {
+        typealias CompletionResult = (Result<Bool, CoreSdkClient.SalemoveError>) -> Void
+        var environment: Environment
```

Comment:
do we have any specific reason to keep `environment` as `var`?

Reply:
Not really, I kept them as var because of my struggles with the `Queue` topic I talked about earlier in the week. However moving this to let and providing init should also be fine. Applies to the Environment conversation below too.

Reply:
Fixed fbb0d8ef7636a0daeeaf95d166b4084fca2f42f9